src/hdf5_time_series_storage.jl

```diff
@@ -76,7 +76,7 @@ function from_file(
     else
         file_path, io = mktemp()
         close(io)
-        cp(filename, file_path; force = true)
+        run(`cp -f $(filename) $(file_path)`)
```

Comment:
This should fail on some Windows systems.  If users have installed GNU coreutils then it will work.  I think it would be safer to check the platform.  I'm not sure what is installed on the CI systems.

```
if Sys.iswindows()
    run(`copy /Y $filename $file_path`)
else
    run(`cp -f $(filename) $(file_path)`)
end
```
It would also be better to put this in a function.  And add a comment describing the Julia bug (with a link) so that future devs can check on removing it.

Reply:
The proposed substitution failed in windows.

Reply:
The final solution was to use a try/catch with a call to cmd before the copy command. I tested the command ``run(`cmd \c copy $src $dst)`)`` in the previous CI run and it works